mpas_analysis/analysis_task_template.py

```diff
@@ -34,11 +34,10 @@
    to be generated and is set up properly.
 
 Don't forget to remove this docstring. (It's not needed.)
-
-Authors
--------
-Xylar Asay-Davis
 '''
+# Authors
+# -------
+# Xylar Asay-Davis
```

Comment:
@pwolfram, here's an example of what I've changed.  Does this work for you?

Reply:
Great!

Reply:
Having this information around is helpful during development but I agree we don't need it in the docs.